### PR TITLE
refactor(dropdown, dropdown-group, dropdown-item): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

**Comment by julio8a** [comment]
@paulcpederson or @jrowlingson, 

can you review this one, it's all individual dropdown PR's in one. 

@bstifle and I already reviewed the design through local build and it's good to go.

**Comment by julio8a** [comment]
The screener bugs are so weird. I merged master into this, not sure why it's showing changes.

File: src/components/calcite-dropdown/calcite-dropdown.scss [hunk 1]
```diff
@@ -1,77 +1,59 @@
 :host {
-  position: relative;
-  display: inline-flex;
-  flex-grow: 1;
+  @apply relative flex-grow;
```
Comment (caripizza) [inline src/components/calcite-dropdown/calcite-dropdown.scss:2]: @julio8a - missing `inline-flex` here - I think that's why those tests are failing

File: src/components/calcite-dropdown/calcite-dropdown.scss [hunk 2]
```diff
@@ -1,77 +1,59 @@
...
-
 :host([active]) .calcite-dropdown-wrapper {
   @include popperWrapperActive();
 }
-
 :host .calcite-dropdown-content {
-  background: var(--calcite-ui-foreground-1);
+  @apply bg-foreground-1
+    overflow-hidden
+    overflow-x-auto
```
Comment (caripizza) [inline src/components/calcite-dropdown/calcite-dropdown.scss:20]: `overflow-y-auto` (unless `overflow-x-auto` is intentional?)

**Review by caripizza — CHANGES_REQUESTED** [review]

**Review by bstifle — APPROVED** [review]

File: src/components/calcite-dropdown/calcite-dropdown.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-dropdown/calcite-dropdown.scss:2]: Jack had suggested it wasn't needed, but I guess it was.

File: src/components/calcite-dropdown/calcite-dropdown.scss (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-dropdown/calcite-dropdown.scss:20]: good catch!

**Pull request merged**